template-helpers/breach-detail.js

```diff
@@ -66,10 +97,8 @@ function compareBreachDates(breach) {
   return false;
 }
 
-function tempOverview(breach, args) {
-  return `This is a make-believe breach overview for the ${breach.Title} breach, which occurred 
-  on ${prettyDate(breach.BreachDate, args)} and was added to our database on ${prettyDate(breach.AddedDate, args)}. 
-  I am waiting for an epiphany as to how these should be localized.`;
+function tempOverview(breach, locales) {
+  return `On June 1, 2018, ${breach.Title} suffered a breach. Once the breach was discovered and verified it was added to our database on ${prettyDate(breach.AddedDate, locales)}.`;
```

Comment:
Should "June 1, 2018" be a variable? (ie: `${prettyDate(breach.BreachDate, locales)}` or something)